### DISCUSSION TITLE
part-ii/2014-99/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#351] [Question comments]
Part II 2014 Number Theory: Paper 3, Section I, $1 F$ 

https://questions.tripos.org/part-ii/2014-99/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The continued fraction calculation is fairly routine. We have that $7^2 = 49 < 51 < 64 = 8^2$, so $\lfloor \sqrt{51} \rfloor = 7 = a_0$. Next, $\frac{1}{\sqrt{51} - 7} = \frac{\sqrt{51} + 7}{2}$, and this has floor $7$ (since $7 < \sqrt{51} < 9$), so we have that $a_1 = 7$. Then $\frac{\sqrt{51} + 7}{2} - 7 = \frac{\sqrt{51} - 7}{2}$, and taking the reciprocal gives $\frac{2}{\sqrt{51} - 7} = \sqrt{51} + 7$. This has floor $14 = a_2$, and we get back to where we were after $a_0$ after subtracting $14$, so $\sqrt{51}$ has continued fraction $[a_0,\overline{a_1,a_2}] = [7,\overline{7,14}]$ as required.

Now we calculate the first few convergents. $\frac{p_0}{q_0} = \frac{7}{1}$, and $\frac{p_1}{q_1} = \frac{50}{7}$. In fact we have $50^2 - 51 \cdot 7^2 = 50^2 - (50 + 1)(50 - 1) = 50^2 - (50^2 - 1) = +1$, so a solution $(x,y)$ is $(50,7)$.

There are no such integers. If there were, then we would have $x^2 \equiv -1 \pmod{3}$ (since $51$ is divisible by $3$), but $-1$ is not a quadratic residue $\mod 3$, so this is impossible.

</details>
